### PR TITLE
*: release v2.0.0-rc.1

### PR DESCRIPTION
**Pull request by Gouthamve**
@fabxc @grobie @brian-brazil

---
This change is [Reviewable](https://reviewable.io/reviews/prometheus/prometheus/3306)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+## v2.0.0-rc.1 / 2017-10-17
+
+* [BUGFIX] Fixed Remote-read to not drop the first series.
+* [BUGFIX] Validate recording-rule names.
+* [BUGFIX] Fix several races.
+* [BUGFIX] Only close blocks if there are no iterators accessing it.
+* [FEATURE] Added a warning for time-drift between the browser and the prometheus-server.
```
Comment (grobie) [inline CHANGELOG.md]: Our usual order is: change, feature, enhancement, bugfix.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
+## v2.0.0-rc.1 / 2017-10-17
```
Comment (grobie) [inline CHANGELOG.md]: Without the `v` in the Changelog (don't ask me, was a long discussion ...).

File: CHANGELOG.md (hunk 1 above)
Comment (brancz) [reply CHANGELOG.md]: I've done this repeatedly as well, how about we document this in the "HOWTO cut a new release" document?

Comment (grobie) [reply CHANGELOG.md]: Be my guest :-)

Comment (brancz) [reply CHANGELOG.md]: Done :)

File: CHANGELOG.md (hunk 2 above)
Comment (Gouthamve) [reply CHANGELOG.md]: Oh, the previous RC had a `v`, removed it now.

File: CHANGELOG.md (hunk 1 above)
Comment (Gouthamve) [reply CHANGELOG.md]: Fixed and will keep in mind from next time.

**Comment by Gouthamve** [comment]
@grobie Removed the previous RC, PTAL!

**Pull request merged**